environments/prod/prod.tfvars

```diff
@@ -411,5 +411,69 @@ frontends = [
     custom_domain    = "www.apply-for-probate.service.gov.uk"
     backend_domain   = ["firewall-prod-int-palo-prod.uksouth.cloudapp.azure.com"]
     certificate_name = "apply-for-probate-service-gov-uk"
+  },
+  {
+    name             = "idam-web-public"
+    custom_domain    = "hmcts-access.service.gov.uk"
+    backend_domain   = ["firewall-prod-int-palo-prod.uksouth.cloudapp.azure.com"]
+    certificate_name = "wildcard-prod-platform-hmcts-net"
```

Comment:
Requires certificate name for `hmcts-access.service.gov.uk`

Reply:
done